ipatests/conftest.py

```diff
@@ -83,6 +83,8 @@ def pytest_configure(config):
 
     # always run doc tests
     config.option.doctestmodules = True
+    # pytest.config global is deprecated
+    pytest.ipa_pytestconfig = config
```

Comment:
This is too much of a hack.

Reply:
I Just want to know, what are the disadvantages of attribute setting of third party module?
Anyway, I fixed as you said. Thank you!

Reply:
IMHO the first approach felt like it's against the spirit of pytests' deprecation.